Bottom: adb7dfa131863105668e032acefc6c89ec053b50
Top:    814924afcf965eb4abf2381590363309574c64b2
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-07-24 16:15:39 +0100

Refresh of hid-gtk-rename-x-y-_crosshair-

---

diff --git a/src/hid/gtk/gui.h b/src/hid/gtk/gui.h
index 7538fb5..1253511 100644
--- a/src/hid/gtk/gui.h
+++ b/src/hid/gtk/gui.h
@@ -179,7 +179,7 @@ typedef struct
   gint view_x0,			/* Viewport in PCB coordinates */
     view_y0, view_width, view_height, view_x, view_y;
 
-  gint crosshair_x, y_crosshair_y;
+  gint crosshair_x, crosshair_y;
 }
 GHidPort;
